tests/data/PersonRegGroupDefs.xml

```diff
@@ -1,1086 +1,1508 @@
+        <sdb-sdb_status>
+          <value>E</value>
+          <value>L</value>
+        </sdb-sdb_status>
+       </filter>
+     </realtime>
+    <query>
+SELECT distinct u.uwnetid 
+FROM uwnetid u
+JOIN category c 
+  on c.validation_id = u.validation_id
+  and u.status_code = 30
+  and c.category_code = 2
+  and c.status_code = 1
+JOIN verify sdb_status
+  on sdb_status.validation_id = u.validation_id
+  and sdb_status.verify_code = 127
+  and sdb_status.source_code = 2
+  and sdb_status.verify_data in ('E','L')
+ORDER by u.uwnetid;</query>
+  </group> 
+
+  <group cn="uw_affiliation_graduate-grad">
+    <title>UW Graduate Students - Graduate</title>
+    <gname>UW Graduate Students - Graduate</gname>
+    <classification>restricted</classification>
+    <memberViewers>
+      <viewer>cn="uw_employee"</viewer>
+      <viewer>cn="u_groups_affiliation_student-read"</viewer>
+    </memberViewers>
+    <memberUpdaters>
+      <updater>dns="iam-tools-irws-watcher.cac.washington.edu"</updater>
+    </memberUpdaters>
+    <desc>UW students who are identified as graduate students, including approved on-leave graduate students. This group is automatically maintained in near-real time from UW-IT Identity Registry data. Includes a grace period for recently enrolled students.</desc>
+     <realtime>
+       <filter type="1">
+        <category-status>
+           <value>2-1</value>
+           <value>2-2</value>
+        </category-status>
+        <sdb-sdb_class>
+          <value>08</value>
+        </sdb-sdb_class>
+       </filter>
+     </realtime>
+    <query>
+SELECT distinct u.uwnetid
+FROM uwnetid u
+JOIN category c
+  on c.validation_id = u.validation_id
+  and u.status_code = 30
+  and c.category_code = 2
+  and c.status_code != 3
+JOIN verify sdb_class
+  on sdb_class.validation_id = u.validation_id
+  and sdb_class.verify_code = 126
+  and sdb_class.source_code = 2
+  and sdb_class.verify_data = '08' 
+ORDER by u.uwnetid;</query>
+  </group>
+
+  <group cn="uw_affiliation_seattle-student">
+    <title>Current Seattle students</title>
+    <gname>Current Seattle students</gname>
+    <classification>restricted</classification>
+    <memberViewers>
+         <viewer>cn="uw_employee"</viewer>
+         <viewer>cn="u_groups_affiliation_student-read"</viewer>
+    </memberViewers>
+    <memberUpdaters>
+      <updater>dns="iam-tools-irws-watcher.cac.washington.edu"</updater>
+    </memberUpdaters>
+    <desc>A group containing all current students whose primary campus is identified as Seattle.  This group is automatically maintained in near-real time from UW-IT Identity Registry data.</desc>
+     <realtime>
+       <filter type="1">
+        <category-status>
+          <value>1-1</value>
+          <value>2-1</value>
+          <value>18-1</value>
+          <value>19-1</value>
+        </category-status>
+        <sdb-branch>
+          <value>0</value>
+        </sdb-branch>
+        <sdb-sdb_status>
+          <value>E</value>
```

Comment:
I see this "E" is XML annotated as sdb_status, but the SQL query is against "verify_data"

Above "0" is XML annotated as branch, but again the SQL query is against "verify_data"

Is there a rhyme or reason to this? How would one understand?